README.md

```diff
@@ -1,50 +1,160 @@
+appium driver install --source=npm appium-xcuitest-driver[@<version>]
+# To install a driver from a local folder (useful for developers)
+appium driver install --source=local /Users/me/sources/appium-xcuitest-driver
+# To install a new driver from github (hm, maybe it's time to publish it to NPM?)
+appium driver install --source=github appium/appium-xcuitest-driver
+
+# To list already installed drivers
+appium driver list --installed
+
+# To update a driver (it must be already installed)
+appium driver update xcuitest
+
+# To uninstall a driver (it won't last forever, wouldn't it?)
+appium driver uninstall xcuitest
+```
+
+#### Drivers Maintained By Appium Team
+
+These drivers are supported by Appium core members. Please report any issues
+or suggestions regarding them to the corresponding GitHub issue tracker.
+Refer to the corresponding driver Readme to know more on how to use it and its requirements.
+
+Name | Description
+--- | ---
+[appium-xcuitest-driver](https://github.com/appium/appium-xcuitest-driver) | Allows to automate iOS native and web apps (via hybrid mode). Also supports tvOS. This driver is based on Apple [XCTest framework](https://developer.apple.com/documentation/xctest) and can only use macOS as the host system.
+[appium-uiautomator2-driver](https://github.com/appium/appium-uiautomator2-driver) | Allows to automate Android native and web apps (via hybrid mode). The driver is based on Google's [UiAutomator](https://developer.android.com/training/testing/other-components/ui-automator) framework.
+[appium-espresso-driver](https://github.com/appium/appium-espresso-driver) | Allows to automate Android native apps. The driver is based on Google's [Espresso](https://developer.android.com/training/testing/espresso) framework.
+[appium-mac2-driver](https://github.com/appium/appium-mac2-driver) | Allows to automate macOS native apps. Also supports tvOS. This driver is based on Apple [XCTest framework](https://developer.apple.com/documentation/xctest) and can only use macOS as the host system.
+[appium-windows-driver](https://github.com/appium/appium-windows-driver) | Allows to automate Windows native and UWP apps. This driver is based on Microsoft's [WinAppDriver](https://github.com/microsoft/WinAppDriver) implementation and can only use Windows 10 as the host system.
+[appium-gecko-driver](https://github.com/appium/appium-geckodriver) | Allows to automate Gecko-engine-based browsers, like Firefox on mobile and desktop platforms. Uses [geckodriver](https://github.com/mozilla/geckodriver/releases) command line tool provided by Mozilla for various platforms.
+[appium-safari-driver](https://github.com/appium/appium-safari-driver) | Allows to automate Safari browser on mobile and desktop platforms . Uses the [safaridriver](https://www.manpagez.com/man/1/safaridriver/) command line tool provided by Apple as part of macOS.
+
+#### Drivers Provided By Third Parties
+
+These drivers are supported by different companies or communities.
```

Comment:
maybe we should address something like _that were included in Appium v1_ since so far we won't keep to update all drivers in README. (I think we'll create such a list in docs)

Reply:
you mean simply list third party drivers that were included into Appium1 before by default and not included anymore, like tizen one?